src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -0,0 +1,169 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Commands.Sql.FailoverGroup.Model;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to add Azure Sql Databases into a Failover Group
+    /// </summary>
+    [Cmdlet(VerbsCommon.Add, "AzureRmSqlDatabaseToFailoverGroup", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.Medium)]
+    public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBase
+    {
+        /// <summary>
+        /// The name of the Azure SQL Database FailoverGroup
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipeline = true,
```

Comment:
@ziwa-msft there should only be one parameter per parameter set that has the `ValueFromPipeline` attribute